src/components/animations/CheckmarkAnimation.js

```diff
@@ -1,83 +1,106 @@
 import React from 'react';
-import { View } from 'react-native';
 import RadialGradient from 'react-native-radial-gradient';
 import Animated, {
-  Easing,
-  useAnimatedStyle,
-  useDerivedValue,
-  withSequence,
+  withDelay,
+  withSpring,
   withTiming,
 } from 'react-native-reanimated';
-import { Path, Svg } from 'react-native-svg';
+import { LargeCheckmarkIcon } from '../icons/svg/LargeCheckmarkIcon';
 import { Flex } from '../layout';
 import styled from '@rainbow-me/styled-components';
 
 const Container = styled(Flex)({
+  alignItems: 'center',
+  height: 132,
   justifyContent: 'center',
   width: '100%',
 });
 
-const Circle = styled(Flex)({
+const Circle = styled(Animated.createAnimatedComponent(Flex))({
```

Comment:
Prefer design system:

```tsx
<Box as={Animated.View} alignItems="center" borderRadius={200} height={{ custom: 120 }} justifyContent="center" width={{ custom: 120 }} style={{ overflow: 'hidden' }}> 
```

Note: adhoc styles such as `overflow` isn't a first-class style prop on `Box`, so adhoc styles are typically appended via `style`.

Reply:
`Animated.createAnimatedComponent(Flex)` doesn't maki things native, because `Flex` is not native component.